main-app/app/scripts/services/game-model.js

```diff
@@ -51,6 +65,7 @@
 
             };       //todo: make move.....
 
+            console.log(cyclePlayerChoice());
```

Comment:
Don't leave test code in for pull requests - the master branch should be theoretically deployable.